pkg/arvo/sur/btc-node-hook.hoon

```diff
@@ -1504,10 +1789,119 @@
       ::
         [%verify-tx-out-proof (list @ux)]
     ::  Control
-        [%help ~]
+    ::
+        $:  %get-memory-info
+            used=@ud
+            free=@ud
+            total=@ud
+            locked=@ud
+            chunks-free=@ud
+            chunks-used=@ud
+        ==
+      ::
+        [%get-rpc-info active-commands=(list rpc-command)]
+      ::
+        [%help help=wall]
+      ::
+        [%logging logging-config=(map @t ?)]
+      ::
+        [%stop res=@t]
+      ::
+        [%uptime uptime=@dr]
     ::  Generating
     ::
         [%generate blocks=(list blockhash)]
+      ::
+        [%generate-to-address blockhashes=(list blockhash)]
+    ::  Mining
+    ::
+        $:  %get-block-template
+            version=@ud
+            rules=(list rule)
+            vb-available=(map @t @ud)
+            vb-required=@ud
+            previous-blockhash=blockhash
+            transactions=(list transaction)
+            coinbase-aux=@t
+            coinbase-value=@ud
+            target=blockhash
+            min-time=@da
+            mutable=(list mutable)
+            nonce-range=@t
+            sigop-limit=@ud
+            size-limit=@ud
+            weight-limit=@ud
+            cur-time=@da
+            bits=@ux
+            height=@ud
+            default-witness-commitment=blockhash
+        ==
+        ::
+        $:  %get-mining-info
+            blocks=@ud
+            current-block-weight=@ud
+            current-block-tx=@ud
+            difficulty=@rd
+            network-hash-ps=@rd
+            pooled-tx=@ud
+            chain=network-name
+            warnings=@t
+        ==
+        ::
+        [%get-network-hash-ps hash-ps=@rd]
+        ::
+        [%prioritise-transaction res=?]
+        ::
+        [%submit-block res=@t]
+        ::
+        [%submit-header res=@t]
+    ::  Network
+    ::
+        [%add-node res=null]
+        ::
```

Comment:
I indented this separator comments to the left of the type declaration, following Mark's feedback from here: https://github.com/urbit/urbit/pull/1892#discussion_r366571586 I'm fine with them either way, although I'll have to change a significant amount of them from my other commits... ;)